stock_account_product_run_fifo_hook/hooks.py

```diff
@@ -180,6 +177,8 @@ def _run_fifo_vacuum_new(self, company=None):
                     or r.create_date == svl_to_vacuum.create_date
                     and r.id > svl_to_vacuum.id
                 )
+                if self.env.context.get("use_past_svl", False):
+                    candidates = all_candidates
```

Comment:
I don't see this upstream https://github.com/odoo/odoo/blob/16.0/addons/stock_account/models/product.py#L433

Reply:
It comes from https://github.com/OCA/stock-logistics-workflow/commit/c9136445025eb15807f796cb38d93594561f5ec6